consensus/poet/core/sawtooth_poet/poet_consensus/poet_block_publisher.py

```diff
@@ -278,34 +278,34 @@ def initialize_block(self, block_header):
 
         # Otherwise, we have a current validator registry entry.  In that
         # case, we need to make sure that we are using the same PPK that the
-        # other validators think we are using.  If not, then we need to switch
-        # the PoET enclave to using the correct keys.
+        # other validators think we are using and update the active key in the
+        # key store.
         elif validator_info.signup_info.poet_public_key != \
                 active_poet_public_key:
-            # Retrieve the key state corresponding to the PoET public key and
-            # use it to re-establish the key used by the enclave.  Also update
-            # the active PoET public key.
-            poet_key_state = \
-                self._poet_key_state_store[
-                    validator_info.signup_info.poet_public_key]
-
-            active_poet_public_key = \
-                SignupInfo.unseal_signup_data(
-                    poet_enclave_module=poet_enclave_module,
-                    sealed_signup_data=poet_key_state.sealed_signup_data)
+            active_poet_public_key = validator_info.signup_info.poet_public_key
```

Comment:
Yes.  We need to change it because on line 291, the key is used to retrieve the key state.  The assignment on line 293 is the one that is not necessarily required, but I put it there so that I can do the assert just to make sure that I didn't screw something up in development.